lm_eval/models/huggingface.py

```diff
@@ -182,13 +143,17 @@ def __init__(
                         if torch.cuda.is_available()
                         else torch.device("cpu")
                     )
-            else:
+            else:  # Parallelism managed by accelerate
                 if device != "cuda":
                     eval_logger.info(
                         f"Using `accelerate launch` or `parallelize=True`, device '{device}' will be overridden when placing model."
                     )
                 # TODO: include in warning that `load_in_8bit` etc. affect this too
-                self._device = torch.device(device)
+                self._device = (
+                    self.accelerator.device
+                    if self.accelerator is not None
```

Comment:
```suggestion
                    if hasattr(self, "accelerator")
```

Reply:
I found that this will error out if one runs with no distributed values set (no `accelerate launch`, no `parallelize=True`